gapic/src/main/com/google/gapid/perfetto/views/TraceConfigDialog.java

```diff
@@ -488,8 +488,8 @@ public BasicInputArea(Composite parent, Settings settings, Theme theme,
 
           gpuCountersLabels[0] = createLabel(counterGroup, sGpu.getCounterIdsCount() + " selected");
```

Comment:
How is the CounterIdsCount() updated properly though?

Reply:
It's updated on L499.